_config/blinkr.yaml

```diff
@@ -30,6 +30,8 @@ skips:
   - !ruby/regexp /^https:\/\/docs.jboss.org\/author\/display\/PLINK\/Security\+Token\+Server\+%28STS%29+/
 #    # JDF-770
   - !ruby/regexp /^https:\/\/docs.jboss.org\/author\/display\/PLINK\/SAML\+Authenticators\+\%28Tomcat\%2CJBossAS\%29+/
+#   RHD-1344
+  - !ruby/regexp /^https?:\/\/it-developers-pr.stage.redhat.com\/.*\/video.*/
```

Comment:
This seems quite dangerous as it means we won't be checking any links on any of the video pages. Or am I missing something?